src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -678,7 +680,7 @@
                                    pivot-grouping-key (m/remove-nth pivot-grouping-key))
               {:keys [sheet]}    @workbook-data]
           (when (or (not group)
-                    (= group 0))
+                    (= qp.pivot.postprocess/NON_PIVOT_ROW_GROUP (int group)))
```

Comment:
what does the call to `int` do here? Can it be a double? Is it worth checking for `number?` in the `when` on the line above?

Reply:
In most cases, the value will be an int. But, there was a bug with Oracle dbs where the value was a BigDecimal. 
So, calling int just coerces things into int for simple comparison. The order of checking (not group) first should always guarantee that we don't try `(int nil)`, and the pivot-grouping value is actually set by some code in the pivot query processor code, so we know it'll never be a string. It'll always be a number or nil.